slack_bolt/listener/thread_runner.py

```diff
@@ -185,12 +185,11 @@ def _start_lazy_function(self, lazy_func: Callable[..., None], request: BoltRequ
         copied_request = self._build_lazy_request(request, func_name)
         self.lazy_listener_runner.start(function=lazy_func, request=copied_request)
 
-    @staticmethod
-    def _build_lazy_request(request: BoltRequest, lazy_func_name: str) -> BoltRequest:
-        copied_request = create_copy(request.to_copyable())
-        copied_request.method = "NONE"
```

Comment:
mypy detected this property is not defined in the request class and actually the property is never used